src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CertificateAssetDownloader.cs

```diff
@@ -121,14 +120,6 @@ internal static class CertificateAssetDownloader
             return null;
         }
 
-        [DynamicDependency("#ctor(System.Net.Http.HttpMessageHandler)", "System.Net.Http.HttpClient", "System.Net.Http")]
-        [DynamicDependency("#ctor", "System.Net.Http.SocketsHttpHandler", "System.Net.Http")]
-        [DynamicDependency("#ctor", "System.Net.Http.HttpRequestMessage", "System.Net.Http")]
-        [DynamicDependency("set_PooledConnectionIdleTimeout", "System.Net.Http.SocketsHttpHandler", "System.Net.Http")]
-        [DynamicDependency("RequestUri", "System.Net.Http.HttpRequestMessage", "System.Net.Http")]
-        [DynamicDependency("Send", "System.Net.Http.HttpClient", "System.Net.Http")]
-        [DynamicDependency("Content", "System.Net.Http.HttpResponseMessage", "System.Net.Http")]
-        [DynamicDependency("ReadAsStream", "System.Net.Http.HttpContent", "System.Net.Http")]
```

Comment:
All of these are now identified by the linker because of the helper type annotation?

Reply:
Only 
> [DynamicDependency("#ctor", "System.Net.Http.HttpRequestMessage", "System.Net.Http")]

is now identified by the linker because of the helper type annotation below.

The rest are unnecessary because the linker is able to directly recognize the types and ctors/properties/methods being used in this method. When it can't recognize the Reflection usage, it warns (like it was previously with `httpRequestMessageType`).